### PR TITLE
Fixes for Event Create

### PR DESCRIPTION
**Pull request by mienaikoe**
Some fixes:
- Form split into 2 columns in Desktop, 1 column in mobile
- Moving Location Name and Note to the left column
- Only supplying location name and note after selection on google map
- Wrapping Form in a white panel
- In Events List, adding some space above the date

### CODE REVIEW

File: client/pages/AddEvent/EventMap.css [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
-
-.mapSearchBox {
+  border: none;
+  width: 280px;
   margin: 10px;
-  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.3);
+  font-size: 1rem;
 }
 
 .addEventInput:disabled, .disabledInputs{
```
[the remark this answers is not in the document]
Comment (k3ntako) [reply client/pages/AddEvent/EventMap.css]: I meant `.addEventInput` and `.disabledInputs`.

[the remark this answers is not in the document]
Comment (mienaikoe) [reply client/pages/AddEvent/EventMap.css]: Ahhh yea!

[the remark this answers is not in the document]
Comment (mienaikoe) [reply client/pages/AddEvent/EventMap.css]: fixed

File: client/pages/AddEvent/EventMap.css [hunk 2]
```diff
@@ -30,3 +25,7 @@
 .locationName{
   font-weight: 900;
 }
+
+.hidden{
```
Comment (k3ntako) [inline client/pages/AddEvent/EventMap.css]: Should've noticed this one earlier. I don't think you're using this either.

**Pull request merged**